Resources/Prototypes/Access/wizard.yml

```diff
@@ -0,0 +1,3 @@
+﻿- type: accessLevel
```

Comment:
Add this to the `AccessConfiguratorUniversal` so admins and mappers can set it.
An ID card for the wizard might be useful as well, but that does not has to be part of this PR.